### PR TITLE
Bug fix: displays just the word now instead of whole key (key = [word]_word)

### PR DESCRIPTION
**Pull request by apluscs**

### CODE REVIEW

File: portfolio/src/main/webapp/comments.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ function renderCommentsChart(){
     table.addColumn('number', 'Count');
     
     json.forEach((word) => {
-      table.addRow([word.key.name, word.propertyMap.count]);
+      table.addRow([word.key.name.substring(0, word.key.name.length - 5), word.propertyMap.count]);
```
Comment (acmcarther) [inline portfolio/src/main/webapp/comments.js:35]: What's 5

Comment (apluscs) [reply portfolio/src/main/webapp/comments.js:35]: The length of the end substring l want to remove. The key name for "cat" would be cat_word. l just want "cat".

Comment (acmcarther) [reply portfolio/src/main/webapp/comments.js:35]: name the constant?

Comment (acmcarther) [reply portfolio/src/main/webapp/comments.js]: keySuffixLen = 5 is probably fine, though consider including comment "suffix is '_word'"

**Pull request merged**